### DISCUSSION TITLE
Repeater not working on Android 11

### DISCUSSION THREAD

**Discussion by cbs525** [Mygod/VPNHotspot#212] [Q&A]
Repeater function is greyed out after turning on
![Screenshot_20210417-104601_VPN_Hotspot](https://user-images.githubusercontent.com/82694948/115102956-d93e7c00-9f6b-11eb-9838-494cdef47a0f.png)

Here are the logs. 
[vpnhotspot-1040802893789439225.log](https://github.com/Mygod/VPNHotspot/files/6328667/vpnhotspot-1040802893789439225.log)

**Comment by Mygod** [comment]
Thanks! Could you try turning Wi-Fi off and on again?

**Reply by cbs525** [reply]
I tried turning off and on and still getting the same issue. The repeater is greyed out after turning on  and it turns off when i turn wifi off

**Reply by Mygod** [reply]
In that case I am afraid your device/ROM might not support Wi-Fi direct properly.

**Reply by cbs525** [reply]
> In that case I am afraid your device/ROM might not support Wi-Fi direct properly.

It worked fine on android 10. The problem started with android 11. I'm using pixel experience plus. You think custom rom's causing the issue?

**Reply by Mygod** [reply]
Yes.
